### ISSUE TITLE
Tor Browser: doesn't show search engine thumbnails

### ISSUE THREAD

**Issue by mehlon** [NixOS/nixpkgs#82594]
**Describe the bug**
Tor Browser doesn't show the thumbnail icons for search engines. 

**To Reproduce**
1. Select the search bar in Tor Browser.
2. Type something.

**Expected behavior**
The icons should appear as they do in Firefox on NixOS, and on Tor Browser on other platforms.

**Screenshots**
![torbrowser](https://user-images.githubusercontent.com/13736092/76686581-c648d200-661c-11ea-9419-d98869d77e32.png)

**Additional context**

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.23, NixOS, 20.03beta448.1f99fd2fdbe (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.2`
 - channels(root): `"nixos-20.03beta570.730453919bd, u-20.09pre216928.991bbef6835"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by mehlon** [comment]
This worked in 9.0.5 but has stopped working since. In 9.0.9 there are no search engine thumbnails.
Please reopen.  @flokli

**Comment by flokli** [comment]
@mehlon if this was originally fixed by #82807, a new bug should be opened - the issue is likely a different one this time.
